app/models/concerns/statusable.rb

```diff
@@ -121,8 +129,8 @@ def physical_item_hash(availability)
   end
 
   def online_holdings
-    return nil unless url_fulltext
-    url_fulltext.map do |entry|
```

Comment:
Return empty array instead? Save caller from nil checks/null pointers etc

Reply:
Good point. Change on the way.

Reply:
@mlooney Changes made. Can you re-review?